exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,51 @@
+#include "example.h"
+#include <vector>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
+    const std::vector<std::string> ordinals = {
+        "", "first", "second", "third", "fourth", "fifth", "sixth",
+        "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"
+    };
+    
+    const std::vector<std::string> gifts = {
+        "", "a Partridge in a Pear Tree", "two Turtle Doves", "three French Hens",
+        "four Calling Birds", "five Gold Rings", "six Geese-a-Laying",
+        "seven Swans-a-Swimming", "eight Maids-a-Milking", "nine Ladies Dancing",
+        "ten Lords-a-Leaping", "eleven Pipers Piping", "twelve Drummers Drumming"
+    };
+    
+    std::string result = "On the " + ordinals[day] + " day of Christmas my true love gave to me: ";
```

Comment:
If you follow the suggestion above, this won't work anymore, because `operator+` doesn't do string concatenation of two `const char*` (which `"On the "` and `ordinals[day]` are). Instead:
```suggestion
    auto result = std::string{"On the "} + ordinals[day] + " day of Christmas my true love gave to me: ";
```